client/src/main/java/com/flipkart/flux/client/runtime/LocalContext.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2012-2016, the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.flipkart.flux.client.runtime;
+
+/**
+ * Maintains all local flux related context
+ * @author yogesh.nachnani
+ */
+public class LocalContext {
```

Comment:
Not here.
But can we rename the existing com.flipkart.flux.domain.Context to ExecutionContext or a better name to avoid confusion?